Class-20-(REST).md

```diff
@@ -0,0 +1,25 @@
+# Class 20: REST
+
+## In the prep work this week, the students learned
+
+1. What REST stands for and why it is important.
+1. How to define statelessness.
+1. What resources are and how the format of a resource differs.
+
+## Announcements
+
+1. This is the last class of the program! Take the time to acknowledge all of the hard work the students have put in!
+1. Check with your course manager for any additional announcements.
+
+## Large Group Time (Instructor)
+
+### Class 20 Topics That Require Careful Attention
+1. Review the characteristics of REST. REST is deceptively simple, but so important for the students to understand!
+1. Go over the Adopt-a-Pet examples! Highlight the URL portions of the URLs for different pet searches.
+1. Make sure to leave time to do a temperature check for the previous lesson. Did everyone finish the studio? Is everyone getting the concepts? Class 19 had a lot of material in it.
+
+## Small Group Time: Class 20 Studio (TA Notes)
+
+1. Make sure to get every student through the Spring Security studio! By the end of the class, everyone should have a fully-functioning web application with authentication!
```

Comment:
Similar to above...need to decide what goes in the studio.

Reply:
We didn't have a studio planned for this class so I thought it was going to be a catch-up day.